suites/squid/nvmeof/tier-3_8-1-nvmeof_4-group_8-gw_scale.yaml

```diff
@@ -0,0 +1,519 @@
+# Scale test suite including all new features from 8.1
+# GW group 1 with 8 GWs and 76 subsystems with 1520 namespaces - all new features of 8.1
+# GW group 2 with 8 GWs and 76 subsystems with 1520 namespaces - all new features of 8.1 + set_qos
+# GW group 3 with 8 GWs and 76 subsystems with 1520 namespaces - No Inband , vmware use cases
+# GW group 4 with 8 GWs and 76 subsystems with 1520 namespaces - scale with no huge pages
+# 5 initiator nodes
+# Bare Metal conf TBD
+
+tests:
+  - test:
+      abort-on-fail: true
+      module: install_prereq.py
+      name: install ceph pre-requisites
+  - test:
+      abort-on-fail: true
+      config:
+        verify_cluster_health: true
+        steps:
+          - config:
+              command: bootstrap
+              service: cephadm
+              args:
+                mon-ip: node1
+                registry-url: registry.redhat.io
+                allow-fqdn-hostname: true
+                log-to-file: true
+          - config:
+              command: add_hosts
+              service: host
+              args:
+                attach_ip_address: true
+                labels: apply-all-labels
+          - config:
+              command: apply
+              service: mgr
+              args:
+                placement:
+                  label: mgr
+          - config:
+              command: apply
+              service: mon
+              args:
+                placement:
+                  label: mon
+          - config:
```

Comment:
In this test case, we are deploying OSDs on all available devices, does that mean this scenario considers NVMe collocated with OSDs on all NVMe nodes?
Or are we taking care of that in config file by giving nodes without free disks to deploy OSDs for NVMe, so that they won't be collocated?

Reply:
Same as above. changed it to labels